usaspending_api/data_load/generic_loaders.py

```diff
@@ -0,0 +1,67 @@
+from usaspending_api.data_load.data_load_helpers import setup_load_lists, setup_mass_load_lists
+
+
+def insert_recipient_locations(cursor, load_objects):
+    columns, values = setup_mass_load_lists(load_objects, "recipient_location")
+    recipient_location_sql = "INSERT INTO references_location {} VALUES {} RETURNING location_id;".format(
+        columns, values
+    )
+
+    cursor.execute(recipient_location_sql)
+    return [tup[0] for tup in cursor.fetchall()]
+
+
+def insert_recipients(cursor, load_objects):
+    columns, values = setup_mass_load_lists(load_objects, "legal_entity")
+    recipient_sql = "INSERT INTO legal_entity {} VALUES {} RETURNING legal_entity_id;".format(columns, values)
+
+    cursor.execute(recipient_sql)
+    return [tup[0] for tup in cursor.fetchall()]
+
+
+def insert_place_of_performance(cursor, load_objects):
```

Comment:
Is these load multiple records like the above functions the name should be plural too. Or a new naming pattern:
 `insert_records_into_legal_entity `, `insert_records_into_references_location `, ...